config/sync/dpl_fbs.settings.yml

```diff
@@ -1 +1 @@
-base_url: 'https://fbs-openplatform.dbc.dk'
+baseUrl: 'https://fbs-openplatform.dbc.dk'
```

Comment:
Please change this into snake case. As mentioned [here](https://github.com/danskernesdigitalebibliotek/dpl-cms/pull/250#discussion_r1209895164).